src/ontology/mondo-edit.obo

```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:GARD"}
```

Comment:
This x-ref cannot be from MONDO:GARD. MONDO:GARD means that GARD told us that it is the correct x-ref. Here, you said it was, not GARD. You should add a comment and CC GARD (ie Eric Sid) and let him know about this x-ref.
 
That being said, this x-ref refers to porphyria and not acquired porphyria. It should not be on this term. Was there a reason to add it here?

Reply:
I was reviewing the xrefs for 'inherited porphyria' and seeing which ones were general for porpyria and could be added as MONDO:mondoIsNarrowerThanSource so this new term would have some connections to other vocabs.

Reply:
I am ok with MONDO:mondoIsNarrowerThanSource